stl/inc/mdspan

```diff
@@ -693,7 +693,8 @@ public:
 
     template <class _OtherExtents>
         requires is_constructible_v<extents_type, _OtherExtents>
-    constexpr explicit(extents_type::rank() > 0) mapping(const layout_stride::mapping<_OtherExtents>& _Other) noexcept
+    constexpr explicit(!(extents_type::rank() == 0 && is_convertible_v<_OtherExtents, extents_type>) )
+        mapping(const layout_stride::mapping<_OtherExtents>& _Other) noexcept

```

Comment:
No change requested: `layout_left::mapping` and `layout_right::mapping` vary in their `noexcept // strengthened` vs. `noexcept`, but this matches the Standard's varying `noexcept`, despite the fact that they both have preconditions. I think this should be an LWG issue (but am too lazybusy to report it).